### PR TITLE
add USAA, add workaround for N26 Banking

### PR DESCRIPTION
**Pull request by mfc**

### CODE REVIEW

File: src/data/Plexus.csv [hunk 1]
```diff
@@ -288,7 +288,7 @@ My Spectrum,November,2020,X,X,4,No reported issues
 My Vodafone Italia,December,2020,3,No notifications,X,X
 MySudo,October,2020,1,Unusable,1,Unusable
 myStop Mobile,November,2020,3,No map,4,No reported issues
-N26,August,2020,X,X,2,Impossible to confirm money transfer (Slideout popup doesn't show up)
+N26,December,2020,2,Confirming money transfer requires workaround (need to select "scheduled tranfer" with frequency "once"),2,Confirming money transfer requires workaround (need to select "scheduled tranfer" with frequency "once")
```
Comment (tomkonidas) [inline src/data/Plexus.csv:291]:
```suggestion
N26,December,2020,2,"Confirming money transfer requires workaround (need to select 'scheduled transfer' with frequency 'once')",2,"Confirming money transfer requires workaround (need to select 'scheduled transfer' with frequency 'once')"
```

File: src/data/Plexus.csv [hunk 2]
```diff
@@ -469,6 +469,7 @@ Uber,June,2020,1,Unusable,2,"Add Payment screen not working, map tiles not visib
 Uber Eats,September,2020,3,"Google Play Services Warning, but no effect",3,"Add Payment screen not working, map tiles not visible"
 UPS,November,2020,X,X,4,No reported issues
 Up Banking,November,2020,1,Cannot log in,4,No reported issues
+USAA,December,2020,4No reported issues,X,X
```
Comment (tomkonidas) [inline src/data/Plexus.csv:472]:
```suggestion
USAA,December,2020,4,No reported issues,X,X
```

**Review by tomkonidas — CHANGES_REQUESTED** [review]
CSV file works weirdly if you have a double quotes in between a field, so we need to wrap it all with double quotes.

**Comment by mfc** [comment]
thanks for the clean up! will do in future pull requests.

**Review by tomkonidas — APPROVED** [review]

**Pull request merged**